tests/models/animals/test_animal_model.py

```diff
@@ -0,0 +1,80 @@
+"""Test module for model.py (and associated functionality).
+
+This module tests the functionality of model.py, as well as other bits of code that
+define models based on the class defined in model.py
+"""
+
+from contextlib import nullcontext as does_not_raise
+from logging import INFO
+
+import pytest
+from numpy import datetime64, timedelta64
+
+from tests.conftest import log_check
+from virtual_rainforest.models.animals.animal_model import AnimalModel
```

Comment:
We should try as a rule to put `vr` imports inside the tests that use them - the  import then becomes part of the test (and if it fails, only that test will fail rather than the whole file).

The only thing that is tricky here is when we want to use a custom exception as part of a parameterised test - then we need a top level import of the exception.